File: desktop/src/com/ntnu/game/desktop/DesktopLauncher.java
package com.ntnu.game.desktop;

import com.badlogic.gdx.backends.lwjgl.LwjglApplication;
import com.badlogic.gdx.backends.lwjgl.LwjglApplicationConfiguration;
import com.ntnu.game.HelicopterGame;

public class DesktopLauncher {
    public static void main(String[] arg) {
        LwjglApplicationConfiguration config = new LwjglApplicationConfiguration();
        config.width = HelicopterGame.SCREEN_WIDTH;
        config.height = HelicopterGame.SCREEN_HEIGHT;
        config.title = HelicopterGame.TITLE;
        new LwjglApplication(new HelicopterGame(), config);
    }
}

File: desktop/src/com/ntnu/game/desktop/DesktopLauncher.kt
package com.ntnu.game.desktop

import com.badlogic.gdx.backends.lwjgl.LwjglApplication
import com.badlogic.gdx.backends.lwjgl.LwjglApplicationConfiguration
import com.ntnu.game.HelicopterGame

object DesktopLauncher {
    @JvmStatic
    fun main(arg: Array<String>) {
        val config = LwjglApplicationConfiguration()
        config.width = 350
        config.height = 600
        config.title = "Chop Chop"
        LwjglApplication(HelicopterGame(), config)
    }
}
